chapters/101.rst

```diff
@@ -40,13 +47,24 @@ Flow
 Flows manage the two main search workflows: indexing data (indexing Flow) and then querying it (querying Flow).
 
 A Flow is made up of several sub-tasks, and it manages the states and context of these sub-tasks. The input and output data of Flows are Documents.
+It is a high level concept in Jina, representing a sequence of steps for accomplishing a task.
+
+You can add logic to the flow by using `add` and create parallelization using `needs` method. Once the flow is built you can open it like opening a file in python and then fit data into it.
```

Comment:
**P**ython. Please check all capitalization is correct throughout the file

## Capitalization

Please capitalize first letters of the following (when used to reference project or component):

- Flow
- Pod
- Pea
- Executor
- Driver
- Jina
- Jina Hub
- Jina Dashboard
- Jina Box
- Document
- Python
- Docker

### Note!

This only applies **outside** a code or CLI context. If you're describing Python, YAML, any programming language, or command line input/output, please use the capitalization you would use **in that context**.

##### Right

```python
from jina.flow import Flow
```

##### Wrong

```python
from Jina.Flow import Flow
```

---

Made with [GitHub saved replies](https://github.com/settings/replies)

Reply:
yes sure